composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.4,<8.0-dev",
+        "php": "~5.6.0",
```

Comment:
Out of band discussion: let's keep testing on PHP 7 but note in the README we don't officially support it for now.